omero/sysadmins/windowsmigration.txt

```diff
@@ -0,0 +1,45 @@
+Migration from OMERO 5.2 on Windows to OMERO 5.3 on Linux
+=========================================================
+
+Backup (Windows)
+----------------
+
+- Create a dump of the OMERO SQL database, either using
+  `pgAdmin <https://www.pgadmin.org/>`_ or the :command:`pg_dump` command line
+  program.
```

Comment:
Perhaps give the command here (` sudo -u postgres pg_dump ...`) which will match the command below or in general refer to the backup/restore page.